templates/ui-package/cypress/integration/examples/ng-auth.spec.js

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+
+let config = require('../../fixtures/config');
+
+context('Actions', () => {
+    beforeEach(() => {
+        Cypress.Cookies.defaults({
+            preserve: cookie => true
+        });
+    });
+
+    before(() => {
+        cy.visit(config.url);
+    });
+
+    it('finds text', () => {
+        cy.contains(' Register a tenant Manage tenants ');
```

Comment:
Maybe the test file should be renamed to something other than "ng-auth" along with the example test scenario?